content/en/docs/guides/_index.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Guides
+linkTitle: "Guides and Tutorials"
+type: docs
+layout: home
+weight: 2
+---
+
+<div class="jumbotron">
+  <h1 class="display-3">Real World Usage</h1>
+  <p class="lead">A collection of authored guides to help you address specific real-world scenarios.</p>
+  <hr class="my-4">
+  <p>It uses utility classes for typography and spacing to space content out within the larger container.</p>
+  <p class="lead">
+    <a class="btn btn-primary btn-lg" href="#" role="button">Learn more</a>
```

Comment:
where is this link supposed to go?

Reply:
Do we really need HTML for that?

Reply:
As i mentioned previously, we need to use HTML for a richer experience in all major landing pages, this is one of them.  The content at the top is a simple placeholder.  We will work with marketing to come up with proper messaging for all landing pages.

Meanwhile, I've enabled to show the link of Guides below.